wavedromtikz.py

```diff
@@ -813,7 +813,13 @@ def render_help_lines(wavedrom):
 
 
 def render_wavedrom(wavedrom):
-    return r"%s%s%s" % (TIKZ_HEADER,
+    wavedromString = TIKZ_HEADER
+    try:
```

Comment:
I suppose wavedrom is a Dict (from memory.. I can be wrong)
I thin is just better to use something like:
`hscale=wavedrom["config"].get("hscale", 1)`
Or to be exstra sure:
`hscale=wavedrom.get("config", {}).get("hscale", 1)`